slapp/transforms/transform_pipeline.py

```diff
@@ -88,6 +90,26 @@ class TransformPipelineSchema(argschema.ArgSchema):
         required=False,
         default="192k",
         description="passed as bitrate to imageio-ffmpeg.write_frames()")
+    font_file = argschema.fields.Str(
```

Comment:
you can change `Str` to `InputFile` and then argschema will validate and raise exceptions for you and you will not need your `mm.pre_load` below.

Reply:
Good suggestion, changed it to InputFile